addons/point_of_sale/static/src/js/gui.js

```diff
@@ -184,20 +184,29 @@ var Gui = core.Class.extend({
     close_other_tabs: function() {
         var self = this;
 
+        // avoid IE to close itself
+        var now = +new Date();
```

Comment:
So in the end we need to identify in the event listener if the upcoming localStorage modification comes from itself or some other tab. So we need to generate a unique ID. The date approach works, an alternative would be ```Math.random()```. Both will work although if you prefer this approach maybe instead of:

```javascript
+new Date();
```

and depending on the type conversion we should use instead

```javascript
Date.now();
```

which does the same thing and is more readable.